src/video/cocoa/cocoa_v.mm

```diff
@@ -214,6 +214,13 @@ static void setupApplication()
 	}
 #endif
 
+#if (MAC_OS_X_VERSION_MAX_ALLOWED >= MAC_OS_X_VERSION_10_12)
+	/* Disable the system-wide tab feature, as we only have one window. */
+	if ([ NSWindow respondsToSelector:@selector(allowsAutomaticWindowTabbing) ]) {
+		NSWindow.allowsAutomaticWindowTabbing = NO;
```

Comment:
Some comments found on the internet suggest that the `respondsToSelector:` call will return `true` even if it isn't supported.
The real property setter method should be called `setAllowsAutomaticWindowTabbing:`. If you combine that with `performSelector:`, you don't even need any conditional compilation. I don't have access to 10.12 though, so I can't check if reality agrees.

Reply:
Thanks! I'll take a look and update it.